addons/l10n_ie/__manifest__.py

```diff
@@ -1,24 +1,24 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
-    'name': 'Ireland - Accounting',
-    'icon': '/account/static/description/l10n.png',
+    "name": "Ireland - Accounting",
+    "version": "1.0",
```

Comment:
We can update the version number here.

Reply:
Indeed, I'm switching to 2.0 since it's very major changes ?

Reply:
We can